index.bs

```diff
@@ -1701,6 +1712,41 @@ a [=moment=] |sourceTime|, and a [=duration=] |eventReportWindow|:
     1. Set |lastEnd| to |lastEnd| + |deadline|.
 1. Return |windows|.
 
+To <dfn>parse report windows</dfn> given an [=ordered map=] |map|, a [=moment=] |sourceTime|,
+and a [=list=] of [=event-level report windows=] |default|:
+
+1. Let |defaultEndDuration| be the [=duration from=] |sourceTime| and the [=event-level report window/end=] of the last element of |default|.
+1. If |map|["`event_report_windows`"] does not [=map/exist=], return |default|.
+1. Let |values| be |map|["`event_report_windows`"].
+1. If |values| is not a [=map=], return error.
+1. Let |startDuration| be 0 seconds.
+1. If |values|["`start_time`"] [=map/exists=]:
+    1. Set |start| to the result of applying the <a spec="html">rules for parsing integers</a> with |values|["`start_time`"].
+    1. If |start| is an error, return error.
+    1. If |start| is less than 0, return error.
```

Comment:
Can this use https://wicg.github.io/attribution-reporting-api/#parse-an-optional-64-bit-signed-integer? As is, it's unclear to me whether the JSON field needs to be parsed as an arbitrary-precision integer.

Reply:
Done